docs/operations/metadata-migration.md

```diff
@@ -72,14 +72,14 @@ In the example commands below:
 
 ```bash
 cd ${DRUID_ROOT}
-java -classpath "lib/*" -Dlog4j.configurationFile=conf/druid/cluster/_common/log4j2.xml -Ddruid.extensions.directory="extensions" -Ddruid.extensions.loadList=[\"mysql-metadata-storage\"] -Ddruid.metadata.storage.type=mysql org.apache.druid.cli.Main tools metadata-init --connectURI="<mysql-uri>" --user <user> --password <pass> --base druid
+java -classpath "lib/*" -Dlog4j.configurationFile=conf/druid/cluster/_common/log4j2.xml -Ddruid.extensions.directory="extensions" -Ddruid.extensions.loadList="[\"mysql-metadata-storage\"]" -Ddruid.metadata.storage.type=mysql -Ddruid.node.type=metadata-init org.apache.druid.cli.Main tools metadata-init --connectURI="<mysql-uri>" --user <user> --password <pass> --base druid
```

Comment:
As an aside on this: what we _really_ need is a script in `bin` that hides all this stuff. This is now mature enough that we don't really need to force the user to check that they copy/pasted this command correctly, and filled in the blanks.

Are you up for writing a simple shell script that is the wrapper? Say, `bin/init-metadata`?

Reply:
@paul-rogers filed GH issue for the script update: https://github.com/apache/druid/issues/13738 . Merging @sergioferragut changes to this doc.